tests/test_core.py

```diff
@@ -63,20 +117,35 @@ def test_dataset_errors():
     with pytest.raises(NotImplementedError):
         d.track("asdf")
 
+    with pytest.raises(NotImplementedError):
+        d.multitrack("asdf")
+
     with pytest.raises(NotImplementedError):
         d.load_tracks()
 
+    with pytest.raises(KeyError):
```

Comment:
hmm weird that this would raise a `KeyError`, while `load_tracks` raises a `NotImplementedError`. Seems like a (small) bug. I'd expect them to have the same behavior